src/Core/Label.js

```diff
@@ -157,6 +176,20 @@ class Label extends THREE.Object3D {
             };
             this.offset.right = this.offset.left + width;
             this.offset.bottom = this.offset.top + height;
+
+            if (this.icon) {
+                rect = this.icon.getBoundingClientRect();
+                const x = Math.round(rect.x);
+                const y = Math.round(rect.y);
+                const width = Math.round(rect.width);
+                const height = Math.round(rect.height);
```

Comment:
don't use `const`, compute  directly with `round` and `floor` in `this.iconOffset` properties.

Reply:
I removed the `const`